src/System.Management.Automation/security/Authenticode.cs

```diff
@@ -619,9 +619,9 @@ private static X509Certificate2 GetCertFromChain(IntPtr pSigner)
                 }
             }
 
-            Diagnostics.Assert(((error == 0) && (signature != null)) || (error != 0), "GetSignatureFromWintrustData: general crypto failure");
+            Diagnostics.Assert(error != 0 || signature != null, "GetSignatureFromWintrustData: general crypto failure");
 
-            if ((signature == null) && (error != 0))
+            if (signature == null && error != 0)
```

Comment:
I feel the code is more readable with parentheses back in.  I am not sure what our coding guidelines say.

Reply:
In this area, coding guidelines say - be consistent, but we also ask for no formatting changes mixed with other changes.

---
In reply to: [144931951](https://github.com/PowerShell/PowerShell/pull/5132#discussion_r144931951) [](ancestors = 144931951)

Reply:
I reverted this. I should have keep myself from removing brackets everywhere, sorry.